server/graphql/v1/mutations/orders.js

```diff
@@ -842,8 +842,9 @@ export async function refundTransaction(_, args, req) {
 
   // 1. Verify user permission. User must be either
   //   a. User that created transaction (within 24h) -- Not implemented yet
```

Comment:
Let's remove A because it's been here for quite some time and we're not implementing it.